app/experimenter/experiments/models/nimbus.py

```diff
@@ -270,6 +270,12 @@ def allocate_bucket_range(self):
         )
 
     def can_review(self, reviewer):
+        if (
+            settings.SKIP_REVIEW_ACCESS_CONTROL_FOR_DEV_USER
+            and reviewer.email == settings.DEV_USER_EMAIL
+        ):
+            return True
```

Comment:
Trying to keep this dev flag constrained to just the local dev user email.

Reply:
Oh interesting cool yeah this is very important not just for local testing but also for the integration tests.  If we do it this way that means we don't need to deal with faking multiple accounts in the integration tests. cc @jrbenny35 this means we should be able to do the whole test with one account the way we did for legacy 👍 

Also @lmorchard instead of flagging this on a new setting we could also just flag it on DEBUG to obviate the need for a new setting?

Reply:
Well, I didn't want to tie it DEBUG in general, because I *did* want to enable the access control for *some* of my dev work.

Reply:
Oh interesting, are there times when you're doing dev with DEBUG=False?

Reply:
No, I'm doing dev with DEBUG=True but I want the review access control enabled

Reply:
Oh I see waht you mean you want to shut off while you're deving right yep makes sense 👍